compiler/crates/relay-typegen/src/visit.rs

```diff
@@ -863,6 +881,9 @@ fn visit_actor_change(
         runtime_imports,
         enclosing_linked_field_concrete_type,
     );
+
+    let description: Option<String> = field.description.as_ref().map(|s| s.to_string());
```

Comment:
Let's keep these as Option<StringKey> until we actual go to print them. (this applies to all similar places)